driftermap.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <title>Surface Drifter Map</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <style type="text/css">
+      #map {
+        position: absolute; left: 0; right: 0; top: 0; bottom: 0;
+		background: #4444FF;
+      }
+    </style>
+    <script src="main.js"></script>
+  </head>
+  <body>
+    <div id="map"></div>
+
+    <!-- Async script executes immediately and must be after any DOM elements used in callback. -->
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBk7VbZaTv1WP5TvIVofiCqzp3sCPBM0pg&callback=initMap&map_ids=d315b7ffc2a2258a&libraries=&v=weekly"
```

Comment:
I see that the map says 'For development purposes only'; is that because af the API here?

Reply:
indeed, this is because this key has no billing information associated, so it is restricted to development only.
to lift that, the key needs to be replaced with one that has a billing account

Reply:
OK, but why do we need to get billed? When we previously used the google maps API, this didn't happen. See e.g. https://github.com/adriftICL/adrift/blob/master/static/australia.js. What si different here?